src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -115,12 +110,15 @@ const SortableTable: FC<SortableTableProps> = ({ listOfContributors, push, remov
       onAuthorSelected={authority => {
         const contributor: Contributor = {
           ...emptyContributor,
-          name: authority.name,
           systemControlNumber: authority.systemControlNumber,
           institutions: authority.orgunitids.map(orgunit => ({
             id: orgunit,
             name: orgunit,
           })),
+          identity: {
+            name: authority.name,
+          },
+          sequence: listOfContributors.length, // TODO: Update this when moving elements in table
```

Comment:
kan du lage en task på dette i Jira sånn at vi ikke glemmer det? 
sequence = placement her (tror jeg, men må sjekke nærmere når det blir tid for å se på det igjen)

Reply:
https://unit.atlassian.net/secure/RapidBoard.jspa?rapidView=73&projectKey=NP&modal=detail&selectedIssue=NP-611